diff -uNr sshguard-1.1.old/configure.ac sshguard-1.1/configure.ac
--- sshguard-1.1.old/configure.ac	2008-07-10 04:53:09.000000000 +0200
+++ sshguard-1.1/configure.ac	2008-09-17 10:41:53.000000000 +0200
@@ -87,8 +87,8 @@
         if test x$iptablespath = x
         then
             # if iptables is not in PATH not, use "/sbin" as default path
-            iptablespath=/sbin
-            AC_MSG_WARN([iptables program not in path! Using /sbin as default unless --with-iptables specified])
+            iptablespath=/usr/sbin
+            AC_MSG_WARN([iptables program not in path! Using /usr/sbin as default unless --with-iptables specified])
         fi
         ;;
      pf)
